src/components/profile-view/layout.jsx

```diff
@@ -0,0 +1,2 @@
+export const leftColumnWidth = 4;
```

Comment:
⛏️`.jsx` files are reserved for components, if your file does not have any components but merely constants, you can have this as `.js` file.